src/libecalc/domain/process/compressor/core/train/simplified_train/envelope_extractor.py

```diff
@@ -0,0 +1,132 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+import numpy as np
+from numpy.typing import NDArray
+
+from libecalc.domain.infrastructure.energy_components.legacy_consumer.system.operational_setting import (
+    ConsumerSystemOperationalSettingExpressions,
+)
+
+
+@dataclass
+class OperationalEnvelope:
+    """Envelope of operating conditions across all operational settings for a compressor train.
```

Comment:
```suggestion
    """Envelope of operational conditions across all operational settings for a compressor train.
```

we have called it `operational settings`. Is that wrong? Should we change that to avoid inconsistency? It is also used in YAML ...